### DISCUSSION TITLE
Is lfo a must for dynamic models?

### DISCUSSION THREAD

**Discussion by hyunjimoon** [Data4DM/BayesSD#75] [people; relating]
Aki used loo for birthday problem [here](https://avehtari.github.io/casestudies/Birthdays/birthdays.html), 
From lfo paper and [vignettes](https://cran.r-project.org/web/packages/loo/vignettes/loo2-lfo.html)

> If there were no time dependence in the data or if the focus is to assess the non-time-dependent part of the model, we could use methods like leave-one-out cross-validation (LOO-CV)
![image](https://user-images.githubusercontent.com/30194633/199720177-b44a931a-01ba-40b1-a1c4-b03a3c569742.png)

- feature request for lfo in loo package

**Comment by tomfid** [comment]
Related: https://onlinelibrary.wiley.com/doi/abs/10.1002/sdr.362

Bootstrapping for confidence interval estimation and hypothesis testing for parameters of system dynamics models
[Gokhan Dogan](https://onlinelibrary.wiley.com/action/doSearch?ContribAuthorRaw=Dogan%2C+Gokhan)
First published: 25 January 2008
https://doi.org/10.1002/sdr.362
Citations: [37](https://onlinelibrary.wiley.com/doi/abs/10.1002/sdr.362#citedby-section)
[PDF](https://onlinelibrary.wiley.com/doi/epdf/10.1002/sdr.362)PDF
[Tools](https://onlinelibrary.wiley.com/doi/abs/10.1002/sdr.362#)
[Share](https://onlinelibrary.wiley.com/doi/abs/10.1002/sdr.362#)
Abstract

Methods for confidence interval estimation and hypothesis testing used in the literature and implemented in system dynamics software packages typically assume that the data are normally distributed, not autocorrelated and not heteroskedastic. In dynamic models these assumptions are often violated. Here we propose the bootstrapping method for confidence interval estimation and hypothesis testing in system dynamics models and provide an overview of the issues involved in applying bootstrapping properly in dynamic models. Bootstrapping is a widely used and robust method but its use in system dynamics models is rare. Bootstrapping can handle violations of the maintained assumptions of traditional methods. It is also valid for small samples whereas traditional methods are valid only asymptotically. Another advantage of bootstrapping is that it is a convenient tool for testing hypotheses including complicated functions of parameters (e.g., multiplication, division of parameters). We provide two examples for illustration, one in discrete time and one in continuous time, including experimental data from the beer distribution game and field data from a study of service quality in the banking industry.
